src/components/document-metadata-modal.yml

```diff
@@ -101,3 +101,9 @@ noAvailableRooms:
 publicContextHeader:
   en: Advanced settings
   de: Erweiterte Einstellungen
+draft:
+  en: Mark as draft
+  de: Als Entwurf markieren
+draftInfo:
+  en: Draft documents can only be accessed and edited by the room owner
+  de: Entwurfsdokumente können nur vom Raumbesitzer zugegriffen und bearbeiten werden
```

Comment:
Although "zugreifen" would be the best word, it unfortunately doesn't work grammatically. Next best would be "abrufen":

"Entwurfsdokumente können nur vom Raumbesitzer abgerufen und bearbeiten werden"

Reply:
Thanks for the correction! I've changed it.